geonode/static/geonode/css/base.css

```diff
@@ -7130,6 +7131,12 @@ article.tab-pane {
 .home #showcase p {
   font-size: 18px;
 }
+.home .partner_logo {
```

Comment:
Remote .home from the partner_logo class prefix so that it applies globally

Reply:
Ok